plugins/modules/pfsense_dns_resolver.py

```diff
@@ -0,0 +1,573 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Chris Liu <chris.liu.hk@icloud.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_dns_resolver
+version_added: 0.6.0
+author: Chris liu (@chris-cyliu), Daniel Huss (@danhuss)
+short_description: Manage pfSense DNS resolver (unbound) settings
+description:
+  - Manage pfSense DNS resolver (unbound) settings
+notes:
+options:
+  state:
+    description: Enable/Disable DNS Resolver
+    default: present
+    choices: [ "present", "absent" ]
+    type: str
+  port:
+    description: Listen Port
+    required: false
+    default: null
+    type: int
+  enablessl:
+    description: Enable SSL/TLS Service
+    required: false
+    default: false
+    type: bool
+  sslcertdescr:
```

Comment:
We tend to use just `cert` or `certificate` elsewhere, so let's rename this to `sslcert`.

Reply:
It's been a little while, but I think all three of these were already different from the XML tags they were written to. So a simple find and replace all should do the trick.